ansible_ai_connect/ai/api/views.py

```diff
@@ -1152,105 +1162,69 @@ def __init__(self):
     def post(self, request) -> Response:
         headers = {"Content-Type": "application/json"}
         request_serializer = ChatRequestSerializer(data=request.data)
-        rh_user_org_id = getattr(request.user, "org_id", None)
-
-        data = {}
-        req_query = "<undefined>"
-        req_model_id = "<undefined>"
-        req_provider = "<undefined>"
-        duration = 0
-        operational_event = {}
-
-        try:
-            if not self.chatbot_enabled:
-                raise ChatbotNotEnabledException()
-
-            if not request_serializer.is_valid():
-                raise ChatbotInvalidRequestException()
-
-            req_query = request_serializer.validated_data["query"]
-            req_model_id = (
-                request_serializer.validated_data["model"]
-                if "model" in request_serializer.validated_data
-                else settings.CHATBOT_DEFAULT_MODEL
-            )
-            req_provider = (
-                request_serializer.validated_data["provider"]
-                if "provider" in request_serializer.validated_data
-                else settings.CHATBOT_DEFAULT_PROVIDER
-            )
-            conversation_id = (
-                request_serializer.validated_data["conversation_id"]
-                if "conversation_id" in request_serializer.validated_data
-                else None
-            )
 
-            start = timeit.default_timer()
+        if not self.chatbot_enabled:
+            raise ChatbotNotEnabledException()
 
-            llm: ModelPipelineChatBot = apps.get_app_config("ai").get_model_pipeline(
-                ModelPipelineChatBot
-            )
+        if not request_serializer.is_valid():
```

Comment:
Note: this will later be done from within `AACSAPIView` and we will get a generic exception (not view specific).